### PR TITLE
python311Packages.tabula-py: 2.8.1 -> 2.8.2; fix build

### PR DESCRIPTION
**Pull request by natsukium**
Diff: https://github.com/chezou/tabula-py/compare/refs/tags/v2.8.1...v2.8.2

Changelog: https://github.com/chezou/tabula-py/releases/tag/v2.8.2

ZHF: https://github.com/NixOS/nixpkgs/issues/265948
https://hydra.nixos.org/build/238995645

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 265993` at 5e2a477d run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python310Packages.tabula-py</li>
    <li>python311Packages.tabula-py</li>
  </ul>
</details>

**Comment by wamserma** [comment]
Result of `nixpkgs-review pr 265993` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.tabula-py</li>
    <li>python310Packages.tabula-py.dist</li>
    <li>python311Packages.tabula-py</li>
    <li>python311Packages.tabula-py.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/tabula-py/default.nix [hunk 1]
```diff
@@ -39,11 +36,15 @@ buildPythonPackage rec {
     setuptools-scm
```
Comment (mweinelt) [inline pkgs/development/python-modules/tabula-py/default.nix:36]: Add setuptools, as you're currently relying on setuptools-scm propagating it.

https://github.com/chezou/tabula-py/blob/v2.8.2/pyproject.toml#L2

Comment (natsukium) [reply pkgs/development/python-modules/tabula-py/default.nix:38]: Thanks, done.

**Pull request merged**